### ISSUE TITLE
Better instructions for User dictionaries

### ISSUE THREAD

**Issue by GeekMustHave** [bartosz-antosik/vscode-spellright#88]
I've attempted to use this extension by clicking on the light bulb next to the line number.

I select the "Add 'toc' to the workspace dictionary", nothing happens.

The work 'toc' is still underlined with the red squiggles. 

Same thing when I use the 'user dictionary'
![image](https://user-images.githubusercontent.com/15093825/32418095-4cfa0f18-c231-11e7-9923-9f91b48bad66.png)

For simpletons like me, it would be helpful to define things a little clearer.

ie: "All the extension's commands add words to these main dictionaries" should be before the "There are two main files, user and workspace dictionaries, both contained in spellright.dict files, one located in user settings folder and another in workspace settings folder (.vscode)."

Just a suggestion for improved clarity:

"Spell Right will automatically add words to either the user or workspace dictionaries when you click on the little light bulb next to the line number, then select which dictionary to save the word in.  Just for reference, those dictionaries are stored in the user setting folder and workspace setting folder.  

Words in either of the user dictionaries are considered to be 'correct'. If you decide to have a workspace dictionary those words will be matched first, after that the words in the user dictionary would be matched.  

If you are interested in viewing the dictionaries in the user settings you can use the 'settings' command Files>Settings>Preferences or the [ctrl]+comma keys.  In the upper right corner of the 'Workspace Setting' tab, there is a selection box to let you switch between *USER SETTINGS* and *WORKSPACE SETTINGS*"

**Comment by GeekMustHave** [comment]
Just found out when I close VSCode and restart it the dictionaries seem to be working.

btw - This is a great addon for VScode and the spelling chanllanged appreciate it.

**Comment by bartosz-antosik** [comment]
It looks, after all, like you have shown a bug. It is supposed to react to adding a word to any of these dictionaries instantly, without restart. This is probably unspotted after the very heart of what version 2.0. changed - the multi-root workspaces. let me have a look.

**Comment by bartosz-antosik** [comment]
In the upcoming version the main problem of what you mention will be corrected - once you add the word to the dictionary document will be rechecked using the word.

I have altered the explanations a bit, but I think once adding the word will start working instantaneously it will be a less of a problem.

As a side result you made me thinking and I have found a further inconsistency here which is dating from the very beginning of the extension - once you operate on files without workspace, the "Add (...) to workspace dictionary" menu item should not be added.
